medmodels/treatment_effect/temporal_analysis.py

```diff
@@ -146,6 +132,28 @@ def find_node_in_time_window(
 
     Raises:
         ValueError: If the time attribute is not found in the edge attributes.
+
+    Examples:
+        This function checks if the event "E1" occurred within a time window of 30 days
+        before and after the last exposure to any medication in the group "medications" for
+        the subject "P1":
+
+        .. code-block:: python
+            :linenos:
+
+            find_node_in_time_window(
+                medrecord,
+                subject_index="P1",
+                event_node="E1",
+                connected_group="medications",
+                start_days=-30,
+                end_days=30,
+                reference="last",
+            )
+
+        .. code-block:: python
+
+            >>> True
```

Comment:
Examples should come before `Raises` according to the docstring guide, no?

Reply:
will be defined in #161